test/author.spec.js

```diff
@@ -0,0 +1,26 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+
+const server = require('../server/index');
+
+const { apiURL } = global;
```

Comment:
use environment variables instead of globals. Globals are fine, but let's keep the consistency. Also, it's very important so we can easily change these values in different environments.